GCC 10 build failed

Since GCC 10 enable -fno-common by default motion 4.3.0 fail to build.

With errors like that:
```
/usr/bin/ld: logger.o (symbol from plugin): in function `get_log_type':
(.text+0x0): multiple definition of `caps'; motion.o (symbol from plugin):(.text+0x0): first defined here
/usr/bin/ld: conf.o (symbol from plugin): in function `config_params':
(.text+0x0): multiple definition of `caps'; motion.o (symbol from plugin):(.text+0x0): first defined here
/usr/bin/ld: conf.o (symbol from plugin): in function `config_params':
```
Full build log http://koji.rpmfusion.org/kojifiles/work/tasks/3435/383435/build.log

--- a/src/netcam.h
+++ b/src/netcam.h
@@ -113,7 +113,7 @@ typedef struct netcam_image_buff {
 } netcam_buff;
 typedef netcam_buff *netcam_buff_ptr;
 
-struct netcam_caps {                    /* netcam capabilities: */
+extern struct netcam_caps {                    /* netcam capabilities: */
         unsigned char streaming;        /*  See the NCS_* defines */
         unsigned char content_length;   /*  0 - unsupported     */
 } caps;

--- a/src/translate.c
+++ b/src/translate.c
@@ -10,6 +10,8 @@
 #include "motion.h"
 #include "translate.h"
 
+int nls_enabled;
+
 void translate_locale_chg(const char *langcd){
 #ifdef HAVE_GETTEXT
     /* This routine is for development testing only.  It is not used for

--- a/src/translate.h
+++ b/src/translate.h
@@ -10,7 +10,7 @@
 #ifndef _INCLUDE_TRANSLATE_H_
 #define _INCLUDE_TRANSLATE_H_
 
-int nls_enabled;
+extern int nls_enabled;
 
 #ifdef HAVE_GETTEXT
 #   include <libintl.h>

